pkg/tarmak/cluster/cluster.go

```diff
@@ -338,7 +338,14 @@ func (c *Cluster) ConfigPath() string {
 	return filepath.Join(c.Environment().Tarmak().ConfigPath(), c.ClusterName())
 }
 
+func (c *Cluster) HubName() string {
+	return fmt.Sprintf("%s-%s", c.environment.Name(), clusterv1alpha1.ClusterTypeHub)
+}
+
```

Comment:
This should ideally live in pkg/environment and then called through the environment interface